### ISSUE TITLE
Action required: Update to cert-manager 0.8.0

### ISSUE THREAD

**Issue by alexellis** [openfaas-incubator/ofc-bootstrap#121]
## Proposed change

Update to cert-manager 0.8.0

## Context

I had an email from LetsEncrypt saying that versions < 0.8.0 will be deprecated and blocked by LetsEncrypt from November.

Here is the web version: https://community.letsencrypt.org/t/blocking-old-cert-manager-versions/98753

## Your Environment

Kubernetes

**Comment by alexellis** [comment]
Duplicate of https://github.com/openfaas-incubator/ofc-bootstrap/issues/122
